<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LocalStorageUpgradeDialog</class>
 <widget class="QDialog" name="LocalStorageUpgradeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Local storage requires upgrade</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="introInfoLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="accountInfoLabel"/>
   </item>
   <item>
    <widget class="QLabel" name="shortDescriptionLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="longDescriptionPlainTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="backupLocalStorageCheckBox">
     <property name="text">
      <string>Backup local storage before the upgrade</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="removeLocalStorageBackupAfterUpgradeCheckBox">
     <property name="text">
      <string>Remove backup of local storage after successful upgrade</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="backupLocalStorageLabel">
     <property name="text">
      <string>Creating backup of local storage...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="backupLocalStorageProgressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="restoreLocalStorageFromBackupLabel">
     <property name="text">
      <string>Restoring local storage from backup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="restoreLocalStorageFromBackupProgressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="upgradeProgressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pickAnotherAccountLabel">
     <property name="text">
      <string>Pick &amp;another account:</string>
     </property>
     <property name="buddy">
      <cstring>accountsTableView</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="accountsTableView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
     <item>
      <spacer name="buttonsHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="switchToAnotherAccountPushButton">
       <property name="text">
        <string>Switch to selected account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createNewAccountPushButton">
       <property name="text">
        <string>Create new account</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPatchPushButton">
       <property name="text">
        <string>Upgrade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitAppPushButton">
       <property name="text">
        <string>Quit app</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusBar">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
